### PR TITLE
Removes Rubberduck.API

### PR DESCRIPTION
**Pull request by retailcoder**
@bclothier going to need your help to figure out why it's still trying to process `Rubberduck.API.dll` 🙏

### CODE REVIEW

File: Rubberduck.Deployment/Rubberduck.Deployment.csproj [hunk 1]
```diff
@@ -56,15 +55,7 @@
...
-    <RubberduckPostBuildTask 
-      Config="$(ConfigurationName)"
-      NetToolsDir="$(SdkPath)bin\NETFX 4.6.1 Tools\"
-      WixToolsDir="$(ProjectDir)WixToolset\"
-      SourceDir="$(TargetDir)"
-      TargetDir="$(TargetDir)"
-      ProjectDir="$(ProjectDir)"
-      IncludeDir="$(ProjectDir)InnoSetup\Includes\"
-      FilesToExtract="Rubberduck.dll|Rubberduck.API.dll" />
+    <RubberduckPostBuildTask Config="$(ConfigurationName)" NetToolsDir="$(SdkPath)bin\NETFX 4.6.1 Tools\" WixToolsDir="$(ProjectDir)WixToolset\" SourceDir="$(TargetDir)" TargetDir="$(TargetDir)" ProjectDir="$(ProjectDir)" IncludeDir="$(ProjectDir)InnoSetup\Includes\" FilesToExtract="Rubberduck.dll|Rubberduck.API.dll" />
```
Comment (MDoerner) [inline Rubberduck.Deployment/Rubberduck.Deployment.csproj]: Why the formatting change?

Anyway, this still states that the API should be extracted.

Comment (retailcoder) [reply Rubberduck.Deployment/Rubberduck.Deployment.csproj]: Huh, that wasn't me... gosh and there I was looking for where the `RubberduckPostBuildTask` came from!
Thanks!

File: Rubberduck.sln [hunk 2]
```diff
@@ -95,7 +85,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Rubberduck.Interaction", "R
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Rubberduck.UnitTesting", "Rubberduck.UnitTesting\Rubberduck.UnitTesting.csproj", "{68F55404-32CA-4414-96B8-E284F3B1F846}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Rubberduck.JunkDrawer", "Rubberduck.JunkDrawer\Rubberduck.JunkDrawer.csproj", "{D7A3F7A3-BD90-4845-B098-AC3AFBF0E16F}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Rubberduck.JunkDrawer", "Rubberduck.JunkDrawer\Rubberduck.JunkDrawer.csproj", "{D7A3F7A3-BD90-4845-B098-AC3AFBF0E16F}"
```
Comment (bclothier) [inline Rubberduck.sln]: Is the change intentional & desired?

Comment (bclothier) [reply Rubberduck.sln]: Answer: [apparently so](https://github.com/dotnet/project-system/issues/1821). Other projects uses the same GUID. Don't we love undocumented mystery....

File: Rubberduck.Deployment/Rubberduck.Deployment.csproj [hunk 3]
```diff
@@ -54,9 +54,7 @@
     <CreateProperty Value="$(ProjectDir)$(OutputPath)$(TargetFileName)">
       <Output TaskParameter="Value" PropertyName="TargetAssembly" />
     </CreateProperty>
-
-    <RubberduckPostBuildTask Config="$(ConfigurationName)" NetToolsDir="$(SdkPath)bin\NETFX 4.6.1 Tools\" WixToolsDir="$(ProjectDir)WixToolset\" SourceDir="$(TargetDir)" TargetDir="$(TargetDir)" ProjectDir="$(ProjectDir)" IncludeDir="$(ProjectDir)InnoSetup\Includes\" FilesToExtract="Rubberduck.dll|Rubberduck.API.dll" />
-
+    <!-- RubberduckPostBuildTask goes here -->
```
Comment (bclothier) [inline Rubberduck.Deployment/Rubberduck.Deployment.csproj]: No, you want to keep it. The last argument, `FilesToExtract` is the one that you need to modify.

```
<RubberduckPostBuildTask 
  Config="$(ConfigurationName)" 
  NetToolsDir="$(SdkPath)bin\NETFX 4.6.1 Tools\" 
  WixToolsDir="$(ProjectDir)WixToolset\" 
  SourceDir="$(TargetDir)" 
  TargetDir="$(TargetDir)" 
  ProjectDir="$(ProjectDir)" 
  IncludeDir="$(ProjectDir)InnoSetup\Includes\" 
  FilesToExtract="Rubberduck.dll" />
```

Comment (retailcoder) [reply Rubberduck.Deployment/Rubberduck.Deployment.csproj]: Ah, makes sense, yes

**Pull request merged**